dcpy/lifecycle/ingest/transform.py

```diff
@@ -45,22 +44,12 @@ def to_parquet(config: Config, local_data_path: Path | None = None):
         shutil.rmtree(TMP_DIR)
     TMP_DIR.mkdir()
```

Comment:
Maybe cut these three lines as well? If the tmp dir is more being handled by `run` at this point

Reply:
I actually did cut these lines at first. But when I ran the tests with pytest, I got an error that the folder doesn't exist. Would you recommend to move this code snippet under the tests setup?

Reply:
I think that'd make sense

Reply:
We're in a slightly funky place right now where there's some semi-assumed folder/file structure that could use some clarifying, but this PR is nice and isolated so that can be more taken care of in #773 which is more meant for overall refactoring (and has some overlap with this PR).

Like ideally, test code doesn't need to "know" about some default temporary directory ideally. But that's better than getting errors here or having code that could wipe out the downloaded raw file before it's transformed to parquet

Reply:
I see. What about modifying this snippet as: "check this dir exists. if it does, move on. If it doesn't, create it for me." and keeping it inside of the `transorm.py`? I wonder if there could be a case where you want to use the function alone, outside of `run.py`

Reply:
And yeah, seems fine to make sure that the output folder exists and make it if it doesn't. I just still think ideally that's coming from some sort of function arg (with default perhaps) rather than a hard-coded constant

Reply:
I think I've seen that in your PRs, may steal it for this fn